sdk/resourcemanager/azure-resourcemanager-appservice/pom.xml

```diff
@@ -51,6 +51,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
why?

Reply:
The test cases `canCRUDFunctionApp` and `canCRUDLinuxFunctionApp` can work in `RECORD` and `LIVE` modes, but not in `PLAYBACK` mode. The detail infomation following as URL: 
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3267132&view=results

Reply:
This time `ManagedServiceIdentity` and `UserAssignedIdentity` have been changed to final class.